examples/app-actions/vercel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "functions": {
+    "pages/api/translations.ts": {
```

Comment:
I am not sure this will work with the new files structure. Have you tried running it on vercel?

Reply:
Fixed. Now it's running